hw2/03-count.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>03 - Count</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+  <script defer src="03-count.js"></script>
+</head>
+
+<body class="container-xs bg-primary d-flex justify-content-center align-items-center vw-100 vh-100">
```

Comment:
Using bootstrap classes on the individual elements (main, input div, etc) to center them allows for more flexibility down the line than putting it all on the body. This mainly only applies to bootstrap and not CSS since classes often work together (e.g container and rows). For example, if you later apply a class to a child of the body, it may or may not override the body style.

Reply:
That's an interesting point! I'll move centering classes to `main`.